components/hubspot/sources/new-form-submission/new-form-submission.mjs

```
@@ -21,6 +21,40 @@ export default {
   },
   methods: {
     ...common.methods,
+    async paginate(params, resourceFn, resultType = null, after = null) {
+      let results = null;
+      let maxTs = after || 0;
+      while (!results || params.after) {
+        results = await resourceFn(params);
+        if (results.paging) {
+          params.after = results.paging.next.after;
+        } else {
+          delete params.after;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)